airgun/views/rhai.py

```diff
@@ -0,0 +1,138 @@
+from widgetastic_patternfly import Button, Dropdown
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import (
+    Checkbox,
+    GenericLocatorWidget,
+    ParametrizedView,
+    Table,
+    Text,
+    TextInput,
+    View
+)
+
+from airgun.views.common import BaseLoggedInView
+
+
+class Overview(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+
+    @View.nested
+    class action_summary(View):
+        title = Text(".//h2[@class='title']")
+
+    @View.nested
+    class newest_systems(View):
+        title = Text(".//h2[@class='title']")
+        table = Table(".//section[contains(normalize-space(.),"
+                      "'Newest Systems')]//table")
+
+    @View.nested
+    class optimize_your_expeirience(View):
+        title = Text(".//h2[@class='title']")
+        registered_systems = Text(".//div[@class='chart-metrics']/span")
+        stability_issues = Text(
+            ".//span[normalize-space(.)='View stability issues']"
+            "/../../div[contains(@class, 'count')]")
+        security_issues = Text(
+            ".//span[normalize-space(.)='View security issues']"
+            "/../../div[contains(@class, 'count')]")
+
+
+class Actions(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+    issues = Text(".//div[@class='num']")
+    export_csv = Button("Export CSV")
+
+
+class InventoryAllHosts(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+    search = TextInput(locator=".//input[@placeholder='Find a system']")
```

Comment:
do you have any reason to use `search` explicitly? Does `SearchableViewMixin` not suffice your needs?

Reply:
It doesn't work on RHAI pages, they developed by different people that didn't follow Satellite ui.